src/daft-core/src/array/ops/sort.rs

```diff
@@ -62,11 +62,16 @@ where
     T: DaftIntegerType,
     <T as DaftNumericType>::Native: Ord,
 {
-    pub fn argsort<I>(&self, descending: bool) -> DaftResult<DataArray<I>>
+    pub fn argsort<I>(&self, descending: bool, nulls_first: bool) -> DaftResult<DataArray<I>>
     where
         I: DaftIntegerType,
         <I as DaftNumericType>::Native: arrow2::types::Index,
     {
+        if nulls_first != descending {
+            return Err(DaftError::NotImplemented(
+                "nulls_first is not implemented".to_string(),
```

Comment:
could be useful to make a function or macro that that returns a clearer error message and maybe link to the github issue.

```
fn ensure_nulls_first(descending, nulls_first) -> DaftResult<()>;
```
then you can call it here

```
ensure_nulls_first(nulls_first, descending)?
```

This will also make it much easier to find all the places we have to update the kernel